pycbc/events/coinc_rate.py

```diff
@@ -62,6 +61,38 @@ def multiifo_noise_coinc_rate(rates, slop):
     return expected_coinc_rates
 
 
+def combination_noise_coinc_rate(rates, slop):
+    """
+    Calculate the expected rate of noise coincidences for a combination of
+    detectors
+
+    Parameters
+    ----------
+    rates: dict
+        Dictionary keyed on ifo string
+        Value is a sequence of single-detector trigger rates, units assumed
+        to be Hz
+    slop: float
+        time added to maximum time-of-flight between detectors to account
+        for timing error
+
+    Returns
+    -------
+    combo_coinc_rate: numpy array
+        Value is expected coincidence rate in the combination, units Hz
+    """
+    # extract ifo list and rates vectors from the dictionary
+    ifos = numpy.array(rates.keys())
```

Comment:
does ifos have to be a numpy array? the noise coincident area function says it takes a list of ifos. 

similarly, does it work to just use rates.values() for the rate vectors?

Reply:
You're right, `ifos` doesn't have to be a numpy array

Aslo, `rates.values()` works and I have updated

Testing this on my laptop rather than the cluster (by accident) has shown that the behaviour of `dictionary.keys()` and `dictionary.values()` will change in python 3; in python 3 they give specific `dict_keys` and `dict_values` objects (tagging @spxiwh in this as he has been working on python 3 updates)

As such, for future-proofing, I have included a `list()` wrapper around the `.keys()` call, which does nothing in python 2 but changes them back to being a list in python 3
The `dictionary.values()` being a `dict_values` object rather than list doesn't matter in this case as it is being looped over and acts in exactly the same way

I have done the same on line 39 to convert the keys to a list

Reply:
OK. It looks like `list(mydict)` would be the same as `list(mydict.keys())` so that's one less pair of brackets.  I'll edit this in the browser and then I think we're finished for this iteration